sdk/servicebus/service-bus/migrationguide.md

```diff
@@ -16,6 +16,12 @@ The new version 7 of the Service Bus library provides the ability to share in so
 
 ## Changes in version 7 of the Service Bus library
 
+### Message format changes
+
+Some key fields have been renamed in the ServiceBusMessage and ServiceBusReceivedMessage to better align with AMQP:
```

Comment:
Also, add a code snippet showing the change in V7 related to the settlement methods that were moved to the receiver?

Reply:
I see a section below where this was mentioned, but no code snippets.

Reply:
```suggestion
Some key fields have been renamed in the ServiceBusMessage and ServiceBusReceivedMessage to better align with [AMQP](https://www.amqp.org/sites/amqp.org/files/amqp.pdf):
```

Reply:
> 
> 
> I see a section below where this was mentioned, but no code snippets.

Can do (we did have the section but code samples are better)

Reply:
Added some code for you @HarshaNalluru  :)